lifecycle-operator/apis/lifecycle/v1beta1/keptntask_types.go

```diff
@@ -55,7 +55,6 @@ type KeptnTaskSpec struct {
 	// Timeout specifies the maximum time to wait for the task to be completed successfully.
 	// If the task does not complete successfully within this time frame, it will be
 	// considered to be failed.
-	// +optional
```

Comment:
why is this not optional anymore?

Reply:
it was doubled, few lines to the bottom there is another // +optional comment

Reply:
ohhhh okok